src/app/header/header.component.ts

```diff
@@ -17,12 +17,13 @@ import { Component } from '@angular/core';
 import { Router } from '@angular/router';
 import { toExtendSite } from '../shared/helpers';
 import { AlertComponent } from '../shared/alert/alert.component';
+import { FlexModule } from '@ngbracket/ngx-layout';
 
 @Component({
   selector: 'app-header',
   templateUrl: './header.component.html',
   standalone: true,
-  imports: [AlertComponent],
+  imports: [AlertComponent, FlexModule],
```

Comment:
The HTML was using flex layout but the module was not included so it had no effect